### PR TITLE
Added a new file Git and PR.md which explains how a beginner can send a PR on gitHUB

### PR DESCRIPTION
**Pull request by kasheravaibhav14**
NOTE:
- Please fill out the template below 

## Did you read the instructions in the [README](https://github.com/the-ethan-hunt/first-timers-guide/blob/master/README.md)?

> Y

## Have you read the instructions in [CONTRIBUTING](https://github.com/the-ethan-hunt/first-timers-guide/blob/master/CONTRIBUTING.md)?

> Y

I have taken inspiration from my senior MANAN GOEL and have written this article on how as a beginner one can send a PR on github.

### CODE REVIEW

**Comment by kasheravaibhav14** [comment]
No changes have been made to pull_request_template.md 
That has been written by mistake.
Please overlook it

File: guides/Version_Control/Git and PR.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Git has been one of the most famous Version control application in this century.
+It works like taking screenshot of project which you are working on and storing it like every version control.
+Git works on local machine and remote servers like github.
+we follow a particular protocol in using git
+**Pull requests**
+    (Basic)(working only in master branch)
+    1. It is a kind of request you make to the owner of the project when you have contributed to his/her project and want that contribution to be shown.
```
Comment (the-ethan-hunt) [inline guides/Version_Control/Git and PR.md]: Your explanation is really great but could you Git SLOC (Lines of code) for operations pulling a PR and committing it?
Include the SLOC and this PR is good to be merged.

**Comment by kasheravaibhav14** [comment]
Thanks!
I'll merge the changes asap

**Comment by kasheravaibhav14** [comment]
I mistakenly deleted the previous PR, but I added a new one so please can
you check it.
Thanks for the patience!

**Pull request closed without merging**